dist/t/spec/features/0040_package_spec.rb

```diff
@@ -37,12 +37,15 @@
       click_link('Home Project')
     end
     click_link('Branch Existing Package')
-    within('#new-package-branch-modal', wait: 5) do # wait for the modal to appear
-      fill_in 'linked_project', with: 'openSUSE.org:openSUSE:Tools'
-      fill_in 'linked_package', with: 'build'
-      click_button('Accept')
+    Capybara.using_wait_time(20) do # wait for the modal to appear
```

Comment:
Why don't you do this in general? Instead of random examples that you've seen fail?

Reply:
There is a part of the tests, at the end of this same spec file, where we iterate over a loop to check for a build result, every `max_wait_time` seconds, a hundred times. I didn't want to change the number of loops.

I find better, that we attach to the max wait time default in general. And only increase it in some special cases, where it makes sense.

Reply:
The "special cases" change depending on which case the test suite is currently executing while it hits resource constraints.

And increasing max wait time will increase the time capybara waits for `page.all('a', class: 'build-state-succeeded')` in the loop but that will decrease the amounts of time the loop will run. The time it takes until `succeed_build.length == 1` will stay the same....

Reply:
I changed the code. I changed the maximum wait time in general.